### DISCUSSION TITLE
GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2

### DISCUSSION THREAD

**Discussion by cyncxl30** [amzn/selling-partner-api-models#3626] [General]
"GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2" is not allowed to be accessed via POST request. The error message "Request for report type 1118 is not allowed at this time" indicates that the request for report type 1118 is currently not permitted.

**Comment by rt3722** [comment]
Facing same issue, either fix this or get payments report to new API

Thank you.

**Reply by usman-rind** [reply]
i am facing the same issue, any solutions?

**Comment by ritikacd** [comment]
Hi @cyncxl30

Settlement reports cannot be requested or scheduled. They are automatically scheduled by Amazon. You can search for these reports using the [getReports](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference#getreports) operation passing reportTypes parameter set to "SETTLEMENT" to check when the last settlement report was available.

Thanks,
Ritika Chaudhary
Selling Partner API Team

**Reply by belalali0626** [reply]
@ritikacd  This produces a error of SETTLEMEN{
    "errors": [
        {
            "code": "InvalidInput",
            "message": "Invalid Report Type _SETTLEMENT_",
            "details": ""
        }
    ]
}

**Reply by ritikacd** [reply]
Hi @belalali0626 

Please use the reportType value based on the version you would like to retrieve: 

- GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE
- GET_V2_SETTLEMENT_REPORT_DATA_XML
- GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2

Thanks,
Ritika
